.gitignore

```diff
@@ -21,4 +21,7 @@ pip-selfcheck.json
 .coverage.*
 .cache
 .pytest*
+.idea
 /build
+/requirements.txt
+/.idea/
```

Comment:
Please remove these changes:
- `.idea` belongs in your system-wide gitignore (PyCharm is your choice, not the project's ;-))
- there's not requirements.txt in this project

Reply:
That's fine, on the other hand it would make life easier to contributors who use idea...